### ISSUE TITLE
pojo-tester does not skip "Setter" check for enums

### ISSUE THREAD

**Issue by niravmsh** [sta-szek/pojo-tester#216]
Having setters in Enums is considered a bad practice and flagged by systems like PMD
There is no way to skip Setter checks on Enum at the moment
Specially when Enums are nested inside other pojos which are been tested for Setters

**Comment by sta-szek** [comment]
I agree, in the future we can skip testing setter for enums by default. For now you can workaround this in simple way:

```
